operation/fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/config/impl/V2ConfigurationImpl.java

```diff
@@ -174,6 +174,7 @@ public boolean isFastExport() {
 
     @Override
     public boolean getStorageProviderUsesRequestAccessToken(String provider) {
-        return FHIRConfigHelper.getBooleanProperty("fhirServer/bulkdata/storageProviders/" + provider + "/requiresAccessToken", Boolean.FALSE);
+        return FHIRConfigHelper.getBooleanProperty("fhirServer/bulkdata/storageProviders/" + provider + "/requiresAccessToken", Boolean.FALSE)
+                && !isStorageProviderHmacPresigned(provider);
```

Comment:
might be worth mentioning in the user guide (section 5 table) that `requiresAccessToken` will have no affect if presigned is true

Reply:
I've added some doc updates there.